thorlcr/activities/indexwrite/thindexwriteslave.cpp

```diff
@@ -241,16 +235,25 @@ class IndexWriteSlaveActivity : public ProcessSlaveActivity, public ILookAheadSt
         {
             if (builder)
             {
-                builder->finish(metadata, &crc, maxRecordSizeSeen);
-                if (!isTLK)
+                // Clear out builder before merging builder stats into inactive stats
+                // so that gatherActiveStatistics doesn't also merge builder stats.
+                Owned <IKeyBuilder> tmpBuilder;
```

Comment:
picky/style: non-standard to have space between Owned/Linked and template type, but I notice it is also out of kilter for the pre-existing 'builder' member.

Reply:
I must have copied it from builder.